.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "no-unused-vars": "off"
```

Comment:
- [ ] Kindly remove the no-unused vars rule, as it doesn't correlate to the linting rules shared by other students. Kindly use only the rules set out [here](https://github.com/microverseinc/linters-config/tree/master/javascript)

Reply:
It's funny that I was told to add that line in the previous project and they claimed it was mandatory. I just like the code reviewers. 😆